plugins/old-omz-completion/old-omz-completion.plugin.zsh

```diff
@@ -0,0 +1 @@
+zstyle ':completion:*' matcher-list 'm:{a-zA-Z}={A-Za-z}' 'r:|[._-]=* r:|=*' 'l:|=* r:|=*'
```

Comment:
This suggestion looks better (and more readable)
https://github.com/robbyrussell/oh-my-zsh/issues/5560#issuecomment-408190809